pytest-embedded-wokwi/pytest_embedded_wokwi/wokwi_cli.py

```diff
@@ -101,9 +102,16 @@ def create_wokwi_toml(self):
 
     def create_diagram_json(self):
         app = self.app
-        diagram_json_path = os.path.join(app.app_path, 'diagram.json')
         target_board = target_to_board[app.target]
 
+        # Check for specific target.diagram.json file first
+        diagram_json_path = os.path.join(app.app_path, (app.target + '.diagram.json'))
+        if os.path.exists(diagram_json_path):
+            shutil.copyfile(diagram_json_path, os.path.join(app.app_path, 'diagram.json'))
```

Comment:
here we're copying the `esp32.diagram.json` to `diagram.json`. what will happen if user has both files? the `diagram.json` will be overwritten.

does wokwi support setting the diagram path?
- if yes, we could use `wokwi --diagram-filepath esp32.diagram.json` directly
- if no, we should backup user's `diagram.json` to `diagram.json.old` first (when exist), then do the copy

Reply:
I will check and update accordingly.

Reply:
There is no argument to specify diagram.json file path.
So I will backup the diagram.json to diagram.json.old. Do we need to restore it afterwards or check also for the .old one?

Reply:
I think the main usage will be just single diagram.json or multiple of them specifying it for each target.
I dont think both options together will be used, it doesn't make much sense.